USAGE.md

```diff
@@ -6,122 +6,118 @@ There are many environment variables to control Box86 behaviour.
 
 #### BOX86_LOG
 Controls the Verbosity level of the logs
- * 0
- * NONE : Current default. No message (except some fatal error)
- * 1
- * INFO : Show some minimum log (like librairies not found)
- * 2
- * DEBUG : Details a lot of stuff (like relocations or function called)
- * 3
- * DUMP : All DEBUG plus DUMP of all ELF Info
+ * 0: NONE : No message (except some fatal error). (Default.)
+ * 1: INFO : Show some minimum log (Example: librairies not found)
+ * 2: DEBUG : Details a lot of stuff (Example: relocations or functions called).
+ * 3: DUMP : All DEBUG plus DUMP of all ELF Info.
```

Comment:
I don't think it's more clear.
Maybe * 0 / NONE; No message.... ?

Reply:
Maybe on a next PR?

Reply:
Yeah, it's definitely something to look into.

I will play around with markdown and see how to make it clearer.